refarch-eai/pom.xml

```diff
@@ -39,12 +39,12 @@
 
         <!-- Testing -->
         <jacoco-maven-plugin.version>0.8.12</jacoco-maven-plugin.version>
-        <argLine></argLine> <!-- Must be empty, definition needed for integration of Jacoco and Surefire via @{argLine} lazy property evaluation -->
+        <argLine /> <!-- Must be empty, definition needed for integration of Jacoco and Surefire via @{argLine} lazy property evaluation -->
     </properties>
 
     <scm>
-        <connection>scm:git:https://github.com/it-at-m/refarch-templates.git</connection>
-        <developerConnection>scm:git:https://github.com/it-at-m/refarch-templates.git</developerConnection>
+        <connection>scm:git:${GITHUB_SERVER_URL}/${GITHUB_REPOSITORY}.git</connection>
```

Comment:
@hupling did you test that somewhere?

Reply:
https://github.com/jan-niklasml/refarch-templates/actions/runs/11211577199/job/31160741696 
@jan-niklasml  und ich haben es getestst
der Fehler der bei der Pip kommt ist ein anderer